<launch>

	<node pkg="visual_behavior_los_ultramarinos" type="ObservadorPelotasHSV" name="ObservadorPelotasHSV" output="screen">
	</node>

	<node pkg="visual_behavior_los_ultramarinos" type="Coordinador" name="Coordinador" output="screen">
	</node>

	<node pkg="visual_behavior_los_ultramarinos" type="Control" name="Control" output="screen">
	</node>

</launch>
